### PR TITLE
docs/useWith: update example

### PR DESCRIPTION
**Pull request by raine**

### CODE REVIEW

File: src/useWith.js [hunk 1]
```diff
@@ -25,9 +25,7 @@ var curry = require('./curry');
  *
  *      var double = y => y * 2;
  *      var square = x => x * x;
- *      var add = (a, b) => a + b;
- *      // Adds any number of arguments together
- *      var addAll = () => R.reduce(add, 0, arguments);
```
Comment (raine) [inline src/useWith.js]: `arguments` is not available with arrow syntax.

**Comment by asaf-romano** [comment]
(...args) is

**Comment by davidchambers** [comment]
Thanks, @raine. These examples will actually be removed as part of #1397, which is very close to being merged. I'd like to close this pull request to avoid introducing merge conflicts unnecessarily.

**Comment by raine** [comment]
No problem, missed that one.

**Pull request closed without merging**